src/views/Nft/market/components/BuySellModals/shared/styles.tsx

```diff
@@ -0,0 +1,28 @@
+import styled from 'styled-components'
+import { Flex, Image } from '@pancakeswap/uikit'
+
+export const RoundedImage = styled(Image)`
```

Comment:
Here live the components that will be also used in Sell modal. Components specific to modals live in their directory's styles.tsx